src/lib/Microsoft.Health.Fhir.Ingest/Service/MeasurementEventNormalizationService.cs

```diff
@@ -88,6 +90,13 @@ private async Task StartConsumer(ISourceBlock<EventData> producer, IAsyncCollect
                             (DateTime.UtcNow - evt.SystemProperties.EnqueuedTimeUtc).TotalMilliseconds);
 
                         var token = _converter.Convert(evt);
+                        var tokenString = JsonConvert.SerializeObject(token);
+                        var megabytes = Encoding.Unicode.GetBytes(tokenString).Length / Math.Pow(1000, 3); // Base 10 definition of MB;
+
+                        _log.LogMetric(
+                            IomtMetrics.DeviceEventIngress(),
+                            megabytes);

```

Comment:
Looks like Azure Monitor recommends using Bytes as the unit. Will update to Bytes